.github/workflows/nightly.yml

```diff
@@ -375,9 +373,56 @@ jobs:
           echo opcache.enable_cli=1 >> /etc/php.d/opcache.ini
           echo opcache.protect_memory=1 >> /etc/php.d/opcache.ini
           echo opcache.jit_buffer_size=1G >> /etc/php.d/opcache.ini
+          echo opcache.jit_max_root_traces=1000000 >> /etc/php.d/opcache.ini
+          echo opcache.jit_max_side_traces=1000000 >> /etc/php.d/opcache.ini
+          echo opcache.jit_max_exit_counters=1000000 >> /etc/php.d/opcache.ini
+          echo opcache.jit_hot_loop=1 >> /etc/php.d/opcache.ini
+          echo opcache.jit_hot_func=1 >> /etc/php.d/opcache.ini
+          echo opcache.jit_hot_return=1 >> /etc/php.d/opcache.ini
+          echo opcache.jit_hot_side_exit=1 >> /etc/php.d/opcache.ini
+          echo memory_limit=-1 >> /etc/php.d/opcache.ini
+          php -v
+      - name: Test PHPSeclib
```

Comment:
Can you please split this into a separate PR? Can you also temporarily copy this to push.yml to show that the tests are succeeding? It would be great if we could trigger this on-demand, but we don't have that currently...

Reply:
Will split this to a separate PR, actually most of the new nightly tests are failing under JIT, might be worth pinging @dstogov about this:

- PHPStan is triggering a JIT assertion on nightly (https://github.com/danog/php-src/actions/runs/6262586495/job/17005155671), opened issue https://github.com/php/php-src/issues/12262
- Amphp with JIT and ext-uv is hanging on nightly (https://github.com/danog/php-src/actions/runs/6261267298/job/17001045071), opened issue https://github.com/php/php-src/issues/12249
- Psalm on 8.2 is triggering a tracing JIT bug (can't run the nightly 8.2 tests on my fork so don't have a link), opened issue https://github.com/php/php-src/issues/12255

Reply:
Split to https://github.com/php/php-src/pull/12270

Reply:
> Will split this to a separate PR, actually most of the new nightly tests are failing under JIT, might be worth pinging @dstogov about this:

Thanks. I've assigned these three issues to myself.